src/components/mission/MissionItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MissionItem = (props) => {
+  const { mission_name, mission_description } = props;
+
+  return (
+    <tr>
+      <td>{mission_name}</td>
+      <td>{mission_description}</td>
+      <td>
+        <button type="button">Active Member</button>
+      </td>
+      <td>
+        <button type="button">Join Mission</button>
+      </td>
+    </tr>
+  );
+};
+
+export default MissionItem;
```

Comment:
Kindly, add prop validations here as react requires them to validate the props being passed down.
For example, you could write, MissionItem.propTypes = {mission_name: PropTypes.string.isRequest, ...};
the ... here represents other attributes to be counted for. Just remember to import PropTypes at the top of your file.